src/js/typeahead/helpers.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+module.exports = {
```

Comment:
or maybe the default but overwritable. The problem here is this is something you'd want to configure based on user research.